03-CSS/src/css/horizon/horizon-game-box.css

```diff
@@ -0,0 +1,51 @@
+body {
+    background: radial-gradient(circle at center, #f3ff00, #1e00ff);
```

Comment:
Das Blau hier beißt sich n bissl mit dem Blau der GameBox.
Wenn du hsl() bzw hsla() Farben verwendest sind Farbtöne leichter abzustimmen.
.gameBox hintergrund in hsl: `hsl(208, 100%, 53%)`
Dann kannst du für den gradient einfach einen dunkleren Ton der gleichen Farbe wählen, z.B. `hsl(208, 100%, 30%)`
Du drehst quasi nur die Helligkeit runter ohne Sättigung oder Farbton zu ändern.
(vgl. aktulle Farbe mit abweichendem Farbton `hsl(247, 100%, 50%)`